src/app/GitUI/CommandsDialogs/BrowseDialog/FormUpdates.cs

```diff
@@ -17,27 +17,31 @@ public partial class FormUpdates : GitExtensionsDialog
     private readonly TranslationString _errorMessage = new("Failed to download an update.");
     #endregion
 
-    public IWin32Window? OwnerWindow;
-    public Version CurrentVersion { get; }
-    public bool UpdateFound;
-    public string UpdateUrl = "";
-    public string NewVersion = "";
+    private IWin32Window? _ownerWindow;
+    private Version _currentVersion;
+    private bool _updateFound;
+    private string _netRuntimeDownloadUrl = string.Empty;
+    private string _updateUrl = string.Empty;
+    private string _newVersion = string.Empty;
+    private Version? _requiredNetRuntimeVersion;
```

Comment:
With recent C# compilers, `string.Empty` has no benefit.

Reply:
It's an old habit.